testsuite/tests/typing-polyvariants-bugs/pr10664a.ml

```diff
@@ -110,7 +110,7 @@ let y = g o;;
 [%%expect{|
 val o : < m : 'a 'c. < n : ([< `A of 'a ] as 'c) -> 'b > > as 'b = <obj>
 val y :
-  < n : _[< `A of '_weak2 ] ->
+  < n : (_[< `A of '_weak2 ] as '_weak3) ->
```

Comment:
Question: do we still want to keep the leading `_` for weak type row variables if we start to always alias them to  `_weak` type variable names?

Reply:
I have no opinion, so up to you or anyone else who chimes in.